components/Loading/AcknowledgeModal.jsx

```diff
@@ -53,10 +58,14 @@ function AcknowledgeModal({onClose}) {
   return (
     <StyledModal open={open}>
       <StyledBox>
-        <Typography variant="h6" sx={{ fontWeight: 500 }} gutterBottom>
+        <Typography variant="h6" gutterBottom>
```

Comment:
I think we're meant to keep `sx={{ fontWeight: 500 }}`, since it usefully overrides the `sx.fontWeight` property.

```suggestion
        <Typography variant="h6" sx={{ fontWeight: 500 }} gutterBottom>
```

Reply:
No, this change was made in the spirit of the issue 🙂

I tried it locally and it looks like the h6 default font weight is 500, so that override isn't necessary either

Reply:
Good catch, then we can consider this resolved!